project/src/pages/not-found-page/not-found-page.tsx

```diff
@@ -0,0 +1,17 @@
+import { Link } from 'react-router-dom';
+import { AppRoute } from '../../const';
+import Header from '../../components/header/header';
+
+import classes from './not-found-page.module.css';
```

Comment:
Лучше использовать общепринятое styles 
```suggestion
import styles from './not-found-page.module.css';
```